File: leetcode/Problem04.cpp
#include <iostream>
#include <vector>
using namespace std; 

class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        for (auto it: nums2) nums1.push_back(it);
        sort(nums1.begin(),nums1.end());
        int siz = nums1.size();
        if (siz % 2) return nums1[siz/2];
        else return (nums1[(siz/2)-1]+nums1[siz/2])/2.0;
    }   
};

File: leetcode/Problem04.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        for i in range(len(nums2)):
            nums1.append(nums2[i])
        size = len(nums1)
        nums1.sort()
        if size % 2 == 1:
            return nums1[size//2]
        else:
            return (nums1[size//2 - 1] + nums1[size//2])/2
